roles/ssh_certificate/tasks/remote-certificate.yml

```diff
@@ -11,13 +11,13 @@
     cert_path: "{{ ssh_cert_path }}"
     public_key_path: "{{ ssh_public_key_path }}"
     private_key_path: "{{ ssh_private_key_path }}"
-    cadn: "{{ ssh_cadn }}"
+    template: "{{ ssh_template }}"
     key_id: "{{ ssh_key_id }}"
     ssh_key_generation_type: "{{ ssh_key_generation_type | default(omit) }}"
     private_key_passphrase: "{{ ssh_private_key_passphrase | default(omit) }}"
     ssh_key_size: "{{ ssh_key_size | default(omit) }}"
     validity_period: "{{ ssh_validity_period | default(omit) }}"
-    policy_dn: "{{ ssh_policy_dn | default(omit) }}"
+    folder: "{{ ssh_policy_dn | default(omit) }}"
```

Comment:
should it be ssh_policy_dn?

Reply:
folder is the name of the variable in the module. The module can be invoked by anyone in any context. The role is simply a shortcut to a set number of tasks that use the given module (in this case, request, retrieve and copy the certificate into the remote host).

ssh_folder is used because the variables file can contain any number of variables for any number of modules or roles (not only the ones for ssh_certificate). So in order to ensure that our folder variable do not override any other variable named "folder" for any other module, the prefix ssh is added to the exposed variable (ssh_certificate is too long), with the "ssh_" prefix, it avoids collision and also gives context of the purpose of the variable (to be used by ssh_certificate module)

Reply:
mmmmm... sorry, I think I was not clear... should it `folder: "{{ ssh_folder | default(omit) }}"` ?